src/plugins/mf_priority.cpp

```diff
@@ -695,13 +695,24 @@ static int validate_cb (flux_plugin_t *p,
         return flux_jobtap_reject_job (p, args, "unable to unpack bank arg");
     }
 
-    // make sure user belongs to flux-accounting DB
+    // make sure user belongs to flux-accounting DB; there are two behaviors
+    // supported in this plugin:
+    //
+    // if the plugin has SOME data about users/banks and the user does not
+    // have an entry in the plugin, the job will be rejected.
+    //
+    // if the plugin has NO data about users/banks and the user does not have
+    // an entry in the plugin, the job will be held until data is received by
+    // the plugin.
     it = users.find (userid);
     if (it == users.end ()) {
-        // user does not exist in internal map yet, so create a bank_info
-        // struct that signifies it's going to be held in PRIORITY
-        add_missing_bank_info (p, h, userid);
-        return 0;
+        if (users.empty ()) {
+            add_missing_bank_info (p, h, userid);
+            return 0;
+        } else {
+            return flux_jobtap_reject_job (p, args,
+                                    "no bank found for user: %i", userid);
```

Comment:
Yeah, @grondo makes a good point here. I just force-pushed a change to make the error message a little more general: `"no bank found for user; please try again later or contact an administrator"`. @ryanday36, let us know if this still looks good to you or if you had something else in mind.

Reply:
I think I liked it better the way it was before 🙃 I could see adding a local patch that gives the user specific information about who to contact for a specific site (i.e. adding LC Hotline contact info here at LLNL (though we can't really do that easily either since TOSS goes to multiple sites)), but I'm not convinced that 'contact an administrator' actually gives the user more info, and 'try again later' is misleading if the user doesn't realize that they need to request a bank.

Reply:
👍 understood! No problem, I'll change it to the original message and force-push. Thanks for the feedback!

Reply:
Thanks @ryanday36, great points.

Reply:
And sorry for the misleading comment @cmoussa1!